users/getUserById.js

```diff
@@ -0,0 +1,22 @@
+const Users = require("./users-model");
+
+function getUserById(req, res) {
+  const { id } = req.params;
+  Users.findById(id)
+    .then(user => {
+      if (user) {
+        res.status(200).json(user);
+      } else {
+        res.status(404).json({
+          message: "Couldn't find user with this id"
```

Comment:
The application client doesn't need to know that you're trying to get the user using Id

Reply:
Done